proxysql-status

```diff
@@ -204,14 +204,21 @@ fi
 
 if [[ $DUMP_ALL -eq 1 || $DUMP_FILES -eq 1 ]]; then
     if [[ -z $TABLE_FILTER ]]; then
+      if [[ -f "/var/lib/proxysql/host_priority.conf" ]]; then
```

Comment:
This only checks for existence, do you also want to check for readability (should use -r)?  Also, can you merge the commit into the v2.0.7-dev branch.